src/YaoExtensions.jl

```diff
@@ -2,32 +2,20 @@ module YaoExtensions
 
 using LuxurySparse
 using Yao, YaoBlocks.ConstGate, BitBasis
```

Comment:
I suggest just rm the dependency of `Yao`, it should make things easier

Reply:
or just use Yao, instead of using `YaoBlocks`, `YaoArrayRegister` etc separately. not necessary to have both of them. our next release of `Yao` will export all there including `YaoSym`